src/utils/index.ts

```diff
@@ -511,6 +515,7 @@ export async function until(condition: MaybeAsync<() => boolean>, timeOutMs = 10
         }
 
         await sleep(pollingIntervalMs) // eslint-disable-line no-await-in-loop
+        wasDone = await Promise.resolve().then(condition) // eslint-disable-line no-await-in-loop
     }
-    return condition()
+    return wasDone
```

Comment:
This can never return `false` anyway right? it'll always either return `true` or `throw` with timeout